src/App.js

```diff
@@ -108,12 +111,21 @@ export default function App() {
 
   if (!loadingStatus.status) {
     return (
-      <>
+      <StyledMain>
         <ResourcesOverview />
-        <Building />
-      </>
+        <span>
```

Comment:
This `span` does not seem to be necessary... Can it be removed?

Reply:
yes the is useless there, is removed